build.gradle

```diff
@@ -70,7 +71,7 @@ allprojects {
     mockitoVersion = '1.10.19'
     javaxServletVersion = '3.1.0'
     guavaVersion = '14.0.1'
-    hiveVersion = '1.2.1.spark2'
+    hiveVersion = '1.21.2.3.1.2.2-1'
```

Comment:
Can we use the one mentioned here? https://github.com/TIBCOSoftware/snappy-spark/pull/210

Reply:
Ok, can either revert this one if you plan on merging that PR, or can change this one. Let me know.